app/controllers/spaces.rb

```diff
@@ -0,0 +1,64 @@
+class MakersBnb < Sinatra::Base
+
+  get '/edit/:space_id' do
+    unless current_user
+      redirect '/sessions/new'
+    end
+    @space = Space.get(params[:space_id])
+    if current_user.id == @space.user_id
+      erb :'spaces/edit'
+    else
+      redirect '/users/account'
+    end
+  end
+
+  post '/edit'do
+    Space.get(params[:space_id]).update(name: params[:name], description: params[:description], price: params[:price])
```

Comment:
You could just write `Space.get(params[:space_id]).update(params)` here to save space